clients/unshaded/src/main/java/tachyon/client/lineage/DummyOutputStream.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.client.lineage;
+
+import java.io.IOException;
+
+import tachyon.client.file.FileOutStream;
+import tachyon.client.file.options.OutStreamOptions;
+
+/**
+ * A dummy output stream that does nothing. This is used for lineage job recomputation. When the
+ * file is not lost on Tachyon, it's no need for the job to rewrite the file.
```

Comment:
`lineage job recomputation` -> `lineage recomputation jop` ?
`it is not needed` or `there is no need`